Web/JQuerry-1/Assignment1/JavaScript.js

```diff
@@ -0,0 +1,15 @@
+function show() {
+				var showValue=document.getElementById('input');
+				showValue.innerHTML="My name is Anshita Arya";
```

Comment:
ACTION: Instead of adding and removing text on show and hide, use style.display = none and style.display= block. Please read about this and implement accordingly.